libs/content/workspace-ffi/src/apple/init.rs

```diff
@@ -24,7 +24,8 @@ pub unsafe extern "C" fn init_ws(
         .create_surface_unsafe(SurfaceTargetUnsafe::CoreAnimationLayer(metal_layer))
         .unwrap();
     let (adapter, device, queue) = pollster::block_on(request_device(&instance, &surface));
-    let format = surface.get_capabilities(&adapter).formats[0];
+    // println!("{:#?}", surface.get_capabilities(&adapter));
```

Comment:
This probably needs to be guarded with some checks, unless you're confident that `formats.len() >= 2`.

Reply:
i'm not 100% confident, so i added a safety check. thanks for the callout